File: prog5.c
#include<stdio.h>

int main()
{
    int grp1;
    printf("Enter the number of people with age below 12: ");
    scanf("%i",&grp1);

    int grp2;
    printf("Enter the number of people with age above (including) 12 and below 18: ");
    scanf("%i",&grp2);

    int grp3;
    printf("Enter the number of people with age above (including) 18 and below 60: ");
    scanf("%i",&grp3);

    int grp4;
    printf("Enter the number of people with age above (including) 60: ");
    scanf("%i",&grp4);

    int amt_grp1=grp1*5;
    int amt_grp2=grp2*10;
    int amt_grp3=grp3*20;
    int amt_grp4=grp4*15;

    int amt=amt_grp1+amt_grp2+amt_grp3+amt_grp4;

    printf("Total amount to be paid: %i",amt);

    return 0;
}

File: prog5.py
g1=int(input("Enter the number of people of age below 12 (12 excluded): "))
g2=int(input("Enter the number of people of age below 18 and above 12 (12 included but 18 excluded): "))
g3=int(input("Enter the number of people of age below 60 and above 18 (18 included but 60 excluded): "))
g4=int(input("Enter the number of people of age above 60 (60 included): "))

amt_g1=g1*5
amt_g2=g2*10
amt_g3=g3*20
amt_g4=g4*15

t_amt=amt_g1+amt_g2+amt_g3+amt_g4

print(f'Total Amount: {t_amt}')